relayer/relay/src/query/client_state.rs

```diff
@@ -0,0 +1,123 @@
+use std::marker::PhantomData;
+use tendermint::rpc::endpoint::abci_query::AbciQuery;
+
+use tendermint::abci;
+
+use relayer_modules::ics02_client::state::ClientState;
+use relayer_modules::ics23_commitment::{CommitmentPath, CommitmentProof};
+use relayer_modules::ics24_host::client::ClientId;
+use relayer_modules::path::{ClientStatePath, Path};
+use relayer_modules::Height;
+
+use super::{ibc_query, IbcQuery, IbcResponse};
+use crate::chain::Chain;
+use crate::error;
+
+pub struct QueryClientFullState<CLS> {
```

Comment:
Is that the corresponding request to `ClientStateResponse`?

Reply:
yes, changed the name of response.